apps/studio/components/to-be-cleaned/Storage/StorageSettings/StorageSettings.tsx

```diff
@@ -84,9 +88,15 @@ const StorageSettings = () => {
     }
   }, [isSuccess, config])
 
-  const formattedMaxSizeBytes = `${new Intl.NumberFormat('en-US').format(
-    STORAGE_FILE_SIZE_LIMIT_MAX_BYTES
-  )} bytes`
+  const maxBytes = useMemo(() => {
```

Comment:
How does this work for free tier? According to our current pricing page free tier is capped at 50MB and other tiers are capped at 50GB. Then, this new feature allows up to 500GB.

Reply:
We're now showing 50 MB for Free Plan